README.md

```diff
@@ -36,6 +32,13 @@ To manage docker as non-root user and verify that you can run
 Document for detailed steps on docker based installation found
 [here](./docs/user/BuildWithDocker.md).
 
+### Prebuilt Binaries
+Document for installing prebuilt binaries courtesy of Precision Innovations found [here](./docs/user/BuildWithPrebuilt.md).
```

Comment:
```suggestion
### Prebuilt Binaries

There are third-party vendors that distribute prebuilt binary versions of the applications required by ORFS. OpenROAD app, Yosys and Klayout. Note that the version you will install from a third party is not guaranteed to work with any version of ORFS. See instructions [here](./docs/user/BuildWithPrebuilt.md).
```

Reply:
@dralabeing please advise on the wording.

Reply:
We are guaranteeing pre-built binaries from Precision but not from other third party vendors. So we should not lump this disclaimer into one category--the user may interpret Precision Innovations to also be third party. Hence need to clarify this point.

The link for pre-built binaries should be referred to as follows:
"To download pre-built binaries for ORFS, refer to the documentation link [here]". Also, retain the note thanking Precision Innovations -- only one instance as discussed.

The disclaimer can then be worded as follows: There are several third-party vendors that distribute pre-built versions of the applications required by ORFS including the OpenROAD application, Yosys and Klayout. Note that the version you will install from a third-party other than Precision Innovations is not guaranteed to work with any version of ORFS. See instructions [here] (./docs/user/BuildWithPrebuilt.md).